source/Misc.hpp

```diff
@@ -975,20 +995,20 @@ typedef struct _OutChanProps
 // Indices for computing output channels:  - customized for the MD_OutParmType
 // approach these are the "QTypes"
 
-const int Time = 0;
-const int PosX = 1;
-const int PosY = 2;
-const int PosZ = 3;
-const int VelX = 4;
-const int VelY = 5;
-const int VelZ = 6;
-const int AccX = 7;
-const int AccY = 8;
-const int AccZ = 9;
-const int Ten = 10;
-const int FX = 11;
-const int FY = 12;
-const int FZ = 13;
+// const int Time = 0;
```

Comment:
Just remove the commented ints

Reply:
I only didn't remove them because above and below there are some other commented out things about the out params that provide some potentially useful context and information. But I'll try and move that documentation parts to more useful locations and remove the commented out code.